mappings/net/minecraft/client/gui/widget/layout/RotatableLayoutWidget.mapping

```diff
@@ -0,0 +1,86 @@
+	METHOD m_wauzovfj copyLayoutSettings ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_ywlwqxpm add (Lnet/minecraft/unmapped/C_dpukplka;Ljava/util/function/Consumer;)Lnet/minecraft/unmapped/C_dpukplka;
+		ARG 1 widget
+		ARG 2 layoutSettingsConsumer
+	CLASS C_ekgfxrpf Orientation
+		METHOD m_fnqdcoug setRotatedY (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;II)V
+			COMMENT Sets the "y" coordinate of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will set the actual y
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will set its x coordinate.
+			ARG 1 widget
+			ARG 2 y
+			ARG 3 height
+		METHOD m_jsvnlxcc getRotatedY (Lnet/minecraft/unmapped/C_dpukplka;)I
+			COMMENT Returns the "y" coordinate of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual y
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will be its x coordinate.
+			ARG 1 widget
+		METHOD m_ltqylntt getRotatedHeight (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;)I
+			COMMENT Returns the height of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual height
+			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL}, it will be
+			COMMENT its width.
+			ARG 1 widget
+		METHOD m_ntoedljy getRotatedWidth (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;)I
+			COMMENT Returns the width of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual width
+			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL}, it will be
+			COMMENT its height.
+			ARG 1 widget
+		METHOD m_otbzsyab getRotatedHeight (Lnet/minecraft/unmapped/C_dpukplka;)I
+			COMMENT Returns the height of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual height
+			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL}, it will be
+			COMMENT its width.
+			ARG 1 widget
+		METHOD m_sfolbtqa getRotatedX (Lnet/minecraft/unmapped/C_dpukplka;)I
+			COMMENT Returns the "x" coordinate of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual x
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will be its y coordinate.
```

Comment:
```suggestion
			COMMENT Returns the X coordinate of the given {@code widget} rotated with this orientation.
			COMMENT <p>
			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual X
			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
			COMMENT it will be its Y coordinate.
```